components/Layout/index.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { ReactNode } from "react";
-import Link from "next/link";
 import Head from "next/head";
+import Header from '../Header';
+import { HEADER_ITEMS } from './constants';
```

Comment:
I think these constants should live in the Header component folder as they apply to the Header component and not necessarily the Layout component.